src/Foundation/NSXpcInterface.cs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2009-2010, Novell, Inc.
+// Copyright 2011, 2012 Xamarin Inc
```

Comment:
The copyrights aren't quite right :smile:

Reply:
What header do you want me to replace it with? I copy/pasted these as-is from another C# file. The copyright headers are all over the place in this repo.

Reply:
Let me get back to you on that.